dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -43,15 +44,16 @@ def setUpClass(cls):
         cls.trained_schema = dp.Profiler(cls.aws_dataset, len(cls.aws_dataset),
                                          profiler_options=profiler_options)
 
-
-    @mock.patch('dataprofiler.profilers.column_profile_compilers.'
+    @mock.patch('dataprofiler.profilers.profile_builder.'
                 'ColumnPrimitiveTypeProfileCompiler')
-    @mock.patch('dataprofiler.profilers.column_profile_compilers.'
+    @mock.patch('dataprofiler.profilers.profile_builder.'
                 'ColumnStatsProfileCompiler')
-    @mock.patch('dataprofiler.profilers.column_profile_compilers.'
+    @mock.patch('dataprofiler.profilers.profile_builder.'
                 'ColumnDataLabelerCompiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.DataLabeler',
+                spec=StructuredDataLabeler)
     def test_add_profilers(self, *mocks):
-        data = pd.DataFrame([1, None, 3, 4, 5, None])
+        data = pd.DataFrame([1, None, 3, 4, 5, None, 1])
```

Comment:
add in another duplicate value to make sure:
`self.assertEqual(5, len(merged_profile.hashed_row_dict))`
didn't change